### ISSUE TITLE
A little Code misspelled?

### ISSUE THREAD

**Issue by Comebaby520** [nikhilroxtomar/MKDCNet#2]
In your model.py file,its  the 150th lines should change the ' s = self.c3(x0) ' to ' s = self.s3(x0) '. I am Right? THANKS

**Comment by innocent-oss** [comment]
> .

Did you implement this? I have few errors

**Comment by Comebaby520** [comment]
Yes,I implemented this.But I didn't meet the errors.Even the s3 written wrong as the c3,the program also could run...
